modules/oe_translation_remote/tests/modules/oe_translation_remote_test/src/Plugin/RemoteTranslationProvider/RemoteOne.php

```diff
@@ -15,4 +18,58 @@
  *   description = @Translation("Remote one translator provider plugin."),
  * )
  */
-class RemoteOne extends RemoteTranslationProviderBase {}
+class RemoteOne extends RemoteTranslationProviderBase {
```

Comment:
This translator is a dummy one that kinda mimics sending the request to an external provider. I use this mostly for testing the flow so far.